server/utils/commonLocale.json

```diff
@@ -31,8 +31,9 @@
     "goalSummaryCard": {
       "info": {
         "aimToAchieveDate": "Aim to achieve this by {{ targetDate }}.",
+        "removedDate": "Removed on {{ lastStatusChangeDate }}",
```

Comment:
can we make this goalRemovedDate to make it consistent with goalAchievedDate please? Or change goalAchievedDate to achievedDate - so long as they follow the same pattern!